src/main/java/svc/models/VIOLATION_STATUS.java

```diff
@@ -0,0 +1,23 @@
+package svc.models;
+
+public enum VIOLATION_STATUS{
+	CONT_FOR_PAYMENT ("Continued for payment"),
+	FTA_WARRANT_ISSUED ("An arrest warrent has been issued for failure to appear to your court date"),
+	DISMISS_WITHOUT_COSTS ("Dismissed"),
+	CLOSED ("Closed");
+	
+	private final String msg;
+	
+	VIOLATION_STATUS (String msg){
+		this.msg = msg;
+	}
+	
+	@Override
+	public String toString(){
```

Comment:
You are probably right that it does serialize the old representations.  But looking at the UI code there are no references to violation.status